pom.xml

```diff
@@ -22,7 +22,7 @@
     </properties>
     <groupId>com.webtrends</groupId>
     <artifactId>wookiee-akka-http_2.11</artifactId>
-    <version>1.6-SNAPSHOT</version>
+    <version>1.7-SNAPSHOT</version>
```

Comment:
We don't need to bump the version on every change. I don't think this warrants a version bump.